test/e2e/apibinding/apibinding_protected_test.go

```diff
@@ -118,3 +119,36 @@ func TestProtectedAPI(t *testing.T) {
 		return resourceExists(resources, "tlsroutes")
 	}, wait.ForeverTestTimeout, time.Millisecond*100, "consumer workspace %q discovery is missing tlsroutes resource", consumerWorkspace)
 }
+
+// TestProtectedAPIFromServiceExports is testing if we can access service exports
+// from root workspace. See https://github.com/kcp-dev/kcp/issues/2184 for more details.
+func TestProtectedAPIFromServiceExports(t *testing.T) {
+	t.Parallel()
+
```

Comment:
I think this got merged while this PR was still in progress, but it would be helpful to add this test to a suite. [See this commit](https://github.com/kcp-dev/kcp/commit/8a3f69b867e48e0e909b2419a17ae3eb4bb46e61) for how to do so.